src/applications/mhv/medications/components/MedicationsList/MedicationsListSort.jsx

```diff
@@ -38,24 +41,45 @@ const MedicationsListSort = props => {
         </VaSelect>
       </div>
       <div className="sort-button">
-        <VaButton
-          uswds
-          type="button"
-          className="va-button"
-          data-testid="sort-button"
-          text="Sort"
-          onClick={() => {
-            if (sortListOption) {
-              sortRxList(sortListOption);
-              history.push(`/?page=1`);
-              waitForRenderThenFocus(
-                "[data-testid='page-total-info']",
-                document,
-                500,
-              );
-            }
-          }}
-        />
+        {showRefillContent ? (
+          <VaButton
+            uswds
+            className="va-button"
+            secondary
+            data-testid="sort-button"
+            text="Sort"
+            onClick={() => {
+              if (sortListOption) {
+                sortRxList(sortListOption);
+                history.push(`/?page=1`);
+                waitForRenderThenFocus(
+                  "[data-testid='page-total-info']",
+                  document,
+                  500,
+                );
+              }
+            }}
+          />
+        ) : (
+          <VaButton
+            uswds
+            type="button"
+            className="va-button"
+            data-testid="sort-button"
+            text="Sort"
+            onClick={() => {
+              if (sortListOption) {
+                sortRxList(sortListOption);
+                history.push(`/?page=1`);
+                waitForRenderThenFocus(
+                  "[data-testid='page-total-info']",
+                  document,
+                  500,
+                );
+              }
+            }}
+          />
+        )}
```

Comment:
those two buttons look identical to me, pretty sure this could be refactored :)

Reply:
thank you! I'll go ahead and do that!